image/docker-compose.yml

```diff
@@ -110,10 +110,7 @@ services:
         max-file: "3"
 
   laboratory:
-    image: kinecosystem/laboratory
-    build:
-      dockerfile: Dockerfile.laboratory
-      context: .
+    image: kinecosystem/laboratory:4ae9ca9
```

Comment:
Now that you removed the build option from the docker-compose, I suggest creating two targets in the Makefile `build-image` to run the `docker build` and `docker tag` commands and `push-image`, to push the latest and the commit hash tag

Reply:
example: https://github.com/kinecosystem/marketplace-server/blob/master/Makefile#L32-L43